src/main/java/org/springframework/data/redis/connection/jedis/JedisConnectionFactory.java

```diff
@@ -249,8 +249,7 @@ public JedisConnectionFactory(RedisSentinelConfiguration sentinelConfiguration,
 			@Nullable JedisPoolConfig poolConfig) {
 
 		this.configuration = sentinelConfiguration;
-		this.clientConfiguration = MutableJedisClientConfiguration
-				.create(poolConfig != null ? poolConfig : new JedisPoolConfig());
+		this.clientConfiguration = MutableJedisClientConfiguration.create(nullSafePoolConfig(poolConfig));
```

Comment:
Inflationary usage of `nullSafe…` methods creates a lot of noise hiding the actual details. Introducing indirections makes it difficult to understand at first sight the details.

Reply:
The lines in question are too busy creating 1) an incomplete statement and 2) unnecessary line break.

Reply:
Also, this is not unlike Spring Framework's own `nullSafeClassName(..)` ([Javadoc](https://docs.spring.io/spring-framework/docs/6.1.0-SNAPSHOT/javadoc-api/org/springframework/util/ObjectUtils.html#nullSafeClassName(java.lang.Object))) `nullSafeConciseToString(..)` ([Javadoc](https://docs.spring.io/spring-framework/docs/6.1.0-SNAPSHOT/javadoc-api/org/springframework/util/ObjectUtils.html#nullSafeConciseToString(java.lang.Object))) methods, or the countless other `nullSafe...` variants.

Your reference to "indirections" is a bit ironic.